packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -130,6 +130,54 @@ export class ERC1155Standard {
     });
   };
 
+  /**
+   * Query for symbol for a given asset.
+   *
+   * @param address - ERC1155 asset contract address.
+   * @returns Promise resolving to the 'symbol'.
+   */
+  getAssetSymbol = async (address: string): Promise<string> => {
+    const contract = new Contract(
+      address,
+      [
+        {
+          inputs: [],
+          name: 'symbol',
+          outputs: [{ name: '_symbol', type: 'string' }],
+          stateMutability: 'view',
+          type: 'function',
+          payable: false,
+        },
+      ],
```

Comment:
[nit] maybe we can add a comment explaining this is the expected abi for the `symbol` method if present on the contract. Same for the `name` method